packages/composables/src/internalHelpers/defaultApiParams.json

```diff
@@ -3,26 +3,30 @@
     "limit": 10,
     "associations": {
       "manufacturer": {},
-      "properties": {
+      "media": {},
+      "productReviews": {},
+      "options": {
```

Comment:
What do we need the `options` for?

Reply:
I got rid of options association param, not everything is based on configurator + optionIds